CondTools/DT/src/DTKeyedConfigHandler.cc

```diff
@@ -554,15 +553,15 @@ void DTKeyedConfigHandler::chkConfigList() {
     brickDataQuery->addToOutputList("CONFIGCMDS.CONFDATA");
     brickDataQuery->setCondition(brickCondition, bindVariableList);
     coral::ICursor& brickDataCursor = brickDataQuery->execute();
-    brickData = new DTKeyedConfig();
-    brickData->setId(brickConfigId);
+    DTKeyedConfig brickData;
+    brickData.setId(brickConfigId);
     while (brickDataCursor.next()) {
       const coral::AttributeList& row = brickDataCursor.currentRow();
-      brickData->add(row["CONFIGCMDS.CONFDATA"].data<std::string>());
+      brickData.add(row["CONFIGCMDS.CONFDATA"].data<std::string>());
     }
-    cond::KeyedElement k(brickData, brickConfigId);
+    cond::KeyedElement k(&brickData, brickConfigId);
     std::cout << "now writing brick: " << brickConfigId << std::endl;
-    outdb->writeOne(k.m_obj, k.m_key, brickContainer);
+    outdb->writeOneIOV(*(k.m_obj), k.m_key, brickContainer);
```

Comment:
```suggestion
    outdb->writeOneIOV((k.m_obj), k.m_key, brickContainer);
```
is this not working like this?

Reply:
No, becaise .cond::KeyedElement::m_obj is a pointer.